connect/runtime/src/main/java/org/apache/kafka/connect/runtime/ConnectorConfig.java

```diff
@@ -94,12 +94,13 @@
 
     public static final String CONFIG_RELOAD_ACTION_CONFIG = "config.action.reload";
     private static final String CONFIG_RELOAD_ACTION_DOC =
-            "The action to take in order to reload changed configuration values (none or restart). "
-                    + "Configuration values will typically only change if using external configuration providers. "
-                    + "A value of 'none' indicates that the new values will not be reloaded. "
-                    + "A value of 'reload' indicates that the new values will be reloaded by restarting the connector. "
-                    + "The restart may actually be scheduled in the future if the external configuration provider "
-                    + "indicates that a configuration value will expire in the future.";
+            "The action that Connect should take on the connector when changes in external " +
+            "configuration providers result in a change in the connector's configuration properties. " +
+            "A value of `none` indicates that Connect will do nothing. " +
```

Comment:
The backtick was from my original comment. 😊 Should change that to single quote like on the next line.